pkg/build/controller/image_change_controller.go

```diff
@@ -33,63 +35,60 @@ type ImageChangeController struct {
 }
 
 // HandleImageRepo processes the next ImageRepository event.
-func (c *ImageChangeController) HandleImageRepo(imageRepo *imageapi.ImageRepository) error {
-	glog.V(4).Infof("Build image change controller detected imagerepo change %s", imageRepo.DockerImageRepository)
-	imageSubstitutions := make(map[string]string)
+func (c *ImageChangeController) HandleImageRepo(repo *imageapi.ImageRepository) error {
+	glog.V(4).Infof("Build image change controller detected imagerepo change %s", repo.Status.DockerImageRepository)
+	subs := make(map[string]string)
 
 	// TODO: this is inefficient
 	for _, bc := range c.BuildConfigStore.List() {
 		config := bc.(*buildapi.BuildConfig)
-		glog.V(4).Infof("Detecting changed images for buildConfig %s", config.Name)
 
-		// Extract relevant triggers for this imageRepo for this config
-		shouldTriggerBuild := false
+		// Extract relevant triggers for this repo for this config
+		shouldBuild := false
 		for _, trigger := range config.Triggers {
 			if trigger.Type != buildapi.ImageChangeBuildTriggerType {
 				continue
 			}
-			icTrigger := trigger.ImageChange
+			change := trigger.ImageChange
```

Comment:
Actually, it's the parameters to an image change trigger. It's the definition of what should change.

----- Original Message -----
> >  		for _, trigger := range config.Triggers {
> >  			if trigger.Type != buildapi.ImageChangeBuildTriggerType {
> >  				continue
> >  			}
> > -			icTrigger := trigger.ImageChange
> > +			change := trigger.ImageChange
> 
> -1, it's not a change, it's a trigger.  specifically an image change trigger.
> 
> 
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1303/files#r26478655